react/week3/todo/src/components/TodoForm.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from "react";
+
+function TodoForm({ todos, setTodos }) {
+  const [inputTodo, setInputTodo] = useState("");
+  const [inputDate, setInputDate] = useState("");
+
+  const buttonAddTodo = () => {
+    const nextTodo = {
+      description: inputTodo,
+      deadline: inputDate,
+    };
+    let newTodos = todos.concat(nextTodo);
+    if (inputTodo && inputDate) setTodos(newTodos);
+  };
+  return (
+    <div>
+      <label>
+        Todo Description:{" "}
```

Comment:
I would still discourage you from applying `{" "}` as a styling element instead of using CSS

Reply:
in VSCode my code formatter always jumps from React to JavaScript when I close the file and reopen. When it hapens I get {" "}. Do you know if there is something I can do about it? :)

Reply:
Did you manage to adjust the VS Code settings? If not, I can try to help you with that :slightly_smiling_face: